Homework_7/homework_07.py

```diff
@@ -0,0 +1,184 @@
+from math import ceil
+
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multi = 1
+
+    # Complete the while loop condition.
+    while True:
+        result = number * multi
+        # десь тут помила, а може не одна
+        if  result > 25:
+            # Enter the action to take if the result is greater than 25
+            break
+        print(f"{number} x {multi} = {result}")
+
+        # Increment the appropriate variable
+        multi += 1
+
+multiplication_table(3)
+# # Should print:
+# # 3x1=3
+# # 3x2=6
+# # 3x3=9
+# # 3x4=12
+# # 3x5=15
+
+
+# task 2
+"""  Написати функцію, яка обчислює суму двох чисел.
+"""
+
+add_two_numbers = lambda num_one, num_two: num_one + num_two
+
+result = add_two_numbers(5, 7)
+print(result)
+
+
+# task 3
+"""  Написати функцію, яка розрахує середнє арифметичне списку чисел.
+"""
+
+def calculate_average(numbers):
+    if not numbers:
+        return None  # Повертає None, якщо список порожній
+    total_sum = sum(numbers)
```

Comment:
It is correct solution. But it is better to just return sum(numbers)/len(numbers) without adding 3 local redundant variables. Less code will be more clear to understand and maintain.